src/client/sandbox/xhr.ts

```diff
@@ -130,9 +130,9 @@ export default class XhrSandbox extends SandboxBaseWithDelayedSettings {
             nativeMethods.xhrOpen.apply(this, arguments);
         });
 
-        overrideFunction(xmlHttpRequestProto, 'send', function () {
+        overrideFunction(xmlHttpRequestProto, 'send', function (this: XMLHttpRequest, ...args: [(string | Document | Blob | ArrayBufferView | ArrayBuffer | FormData | URLSearchParams | ReadableStream<Uint8Array>)?]) {
```

Comment:
We don't process the arguments of the send function, and because we can just write `...args: [any]`